libs/ledger-live-common/.ci.integration.env

```diff
@@ -1,2 +1,3 @@
 DISABLE_MOCKED_WARNING=true
-ONLY_INTEGRATION_TESTS=true
\ No newline at end of file
+ONLY_INTEGRATION_TESTS=true
+API_STELLAR_HORIZON_STATIC_FEE=true
```

Comment:
@gre This is used to get a static recommended fee for Stellar transactions. Otherwise, it will fetch the current value from the public network and tests will fail because the snapshot expects a different value. How should we handle this instead? The check happens [here](https://github.com/LedgerHQ/ledger-live/blob/feat/stellar/libs/ledger-live-common/src/families/stellar/api/horizon.ts#L64:L71).

I'm not sure if this belongs in this file, it used to be in `.ci.env`.